bin/latest-stable

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env bash
 
+# get the directory of the current script
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
 # calling a script to list all elixir releases
 # reject releases before 1.0.0 (starting with 0)
 # reject main and master branches
 # reject releases with otp version
 # reject release candidates
 # reject empty line
 # finally take last one
-ASDF_ELIXIR_LATEST_VERSION=$(./list-all | tr " " "\n" | grep -Ev "^0|^main|^master|otp|rc|^$" | tail -n 1)
+ASDF_ELIXIR_LATEST_VERSION=$($SCRIPT_DIR/list-all | tr " " "\n" | grep -Ev "^0|^main|^master|otp|rc|^$" | tail -n 1)
```

Comment:
I think `$SCRIPT_DIR` here should be wrapped in quotes, otherwise this looks good to me.